src/container/container.controller.ts

```
@@ -0,0 +1,19 @@
+
+@Controller('container')
+export class ContainerController {
+  constructor(private readonly containerService: ContainerService) {}
+
+  @Post('run')
+  async runContainer(@Body() body: any, @Res() res: Response) {
+    try {
+      const result = await this.containerService.runContainer(body);
+      res.json(result);
+    } catch (error) {
+      res.status(500).json({ error: 'Deployment failed', message: error.message });
```

ESLint

[error] Replace `.status(500)` with `⏎········.status(500)⏎········`

(prettier/prettier)